### PR TITLE
New package: fdm at 2.0

### PR DESCRIPTION
**Pull request by tristelune1**
## Description of package

Software to fetch and deliver mail. 

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by dylanaraps** [comment]
Is this patch really needed?

**Comment by tristelune1** [comment]
Yes: [reported issue](https://github.com/nicm/fdm/issues/71)

Because the patch modify `Makefile.am`, it was necessary to add `automake` as a dependency. With the next release, I should be able to simplify the `build` file. 

But probably the next release won't come soon.

**Comment by tristelune1** [comment]
Unfotunately it didn't work. It I understand your idea, I ran `aclocal` and `automake` and did a `diff` on the `Makefile.in`. 

You can have a look at my new patch. I can say it compiles and I have no error if I execute `fdm`. But I didn't test it extensively.

File: community/fdm/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+patch -p1 < musl.patch
+
+./configure \
+	--prefix=/usr \
+	--enable-pcre
```
Comment (dylanaraps) [inline community/fdm/build]: Indentation is 4 spaces.

**Comment by dylanaraps** [comment]
OK. I think it then makes more sense to swap back to how you had it at the start. Apologies...

**Comment by tristelune1** [comment]
No problem, I learnt a little bit better the build system by the way. It should be ok now.

File: community/fdm/build [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 patch -p1 < musl.patch
 
+aclocal
+automake
```
Comment (dylanaraps) [inline community/fdm/build]: Can this simply be: `autoreconf` or `autoreconf -i`?

Comment (tristelune1) [reply community/fdm/build]: `autoreconf` seems to be enough. Done!

**Pull request merged**